### DISCUSSION TITLE
购买了一年的专业版，但是无法接收验证码，无法使用专业版，请解决问题

### DISCUSSION THREAD

**Discussion by noisytomato** [getlantern/lantern-forum-cn#779] [Q&A]
购买了一年的专业版，无法使用专业版，无法收到验证码，发邮件没人理，请给我解决问题，页面全是这样的问题，我只想登录专业版请尽快解决我的诉求 我的邮箱191122609@qq.com
![Uploading 微信图片_20240813130233.jpg…]()

**Comment by noisytomato** [comment]
打开论坛全部是一样的问题跟诉求，蓝灯打底有没有人解决问题

**Comment by MaxMKMax** [comment]
@noisytomato 好的. 我帮您提交问题. 谢谢您支持蓝灯! 非常抱歉影响了您的体验感. 收不到验证码的邮件是因为国内挡住了邮件发送. 所以导致了这一系列的问题. 非常抱歉啊!

**Reply by noisytomato** [reply]
你们解决问题需要多长的时间，我不能无休止等下去，如果没有时限，我就考虑采取其他举措了

**Reply by noisytomato** [reply]
@Mike777Khoo 两天过去毫无进展，这是准备把钱骗了就不管了啊

**Reply by kidokogills** [reply]
请问，意思是不是如果邮箱用gmail，就不会被挡住了？@Mike777Khoo

**Reply by MaxMKMax** [reply]
@noisytomato 马上再次帮您询问了. 您放心, 会帮您解决问题的.

**Reply by MaxMKMax** [reply]
@kidokogills 这是很好的问题. 技术上Gmail不会被墙挡住. 但是也会有种情况就是看地域性的. 也依然会被挡住. 但是Gmail会比较方便, 问题比较少是肯定的.

**Comment by noisytomato** [comment]
谢谢，麻烦尽快解决我的问题！我用蓝灯也很多年了第一次遇见这个事情



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mike K ***@***.***&gt;
发送时间: 2024年8月13日 19:05
收件人: getlantern/lantern-forum-cn ***@***.***&gt;
抄送: noisytomato ***@***.***&gt;, Mention ***@***.***&gt;
主题: Re: [getlantern/lantern-forum-cn] 购买了一年的专业版，但是无法接收验证码，无法使用专业版，请解决问题 (Discussion #779)





 
@noisytomato 好的. 我帮您提交问题. 谢谢您支持蓝灯! 非常抱歉影响了您的体验感. 收不到验证码的邮件是因为国内挡住了邮件发送. 所以导致了这一系列的问题. 非常抱歉啊!

**Comment by noisytomato** [comment]
一周过去了反馈了个寂寞啊，什么回复都没有@Mike777Khoo 到底什么时候能解决我的问题，要不就是告诉我退款如何申请

**Reply by MaxMKMax** [reply]
@noisytomato 您好. 已经给您弄好了. 问题已解决.
